post-install.sh

```diff
@@ -8,4 +8,10 @@ if [ ! -f "$HOME/.local/share/applications/vortex.desktop" ]; then
     ln -s ~/.pikdum/steam-deck-master/vortex/install-vortex.desktop ~/Desktop/install-vortex.desktop || true
 fi
 
+ln -sf ~/.local/share/applications/vortex.desktop ~/Desktop/
+ln -sf ~/.pikdum/steam-deck-master/vortex/skyrim-post-deploy.desktop ~/Desktop/
+ln -sf ~/.pikdum/steam-deck-master/vortex/fallout4-post-deploy.desktop ~/Desktop/
+ln -sf ~/.pikdum/steam-deck-master/vortex/falloutnv-post-deploy.desktop ~/Desktop/
+ln -sf ~/.pikdum/steam-deck-master/vortex/oblivion-post-deploy.desktop ~/Desktop/
```

Comment:
**suggestion:** Maybe this should all be wrapped in a check that `~/.local/share/applications/vortex.desktop` exists, so they don't show up when Vortex isn't installed yet.

Reply:
I can do that. Sorry was a simple edit from my phone.